tests/api_tests/nodes/test_views.py

```diff
@@ -848,6 +848,9 @@ def setUp(self):
 
         self.user_two = AuthUserFactory()
 
+        self.deleted_project = NodeFactory(parent=self.public_project, is_deleted=True, creator=self.user)
```

Comment:
If you only have the one test using this, move these into the test itself (and don't save them on the object).